VirtoCommerce.CatalogModule.Data/packages.config

```diff
@@ -10,8 +10,11 @@
   <package id="System.Collections.Concurrent" version="4.3.0" targetFramework="net461" />
   <package id="System.ComponentModel.Annotations" version="4.4.1" targetFramework="net461" />
   <package id="System.ComponentModel.Primitives" version="4.3.0" targetFramework="net461" />
+  <package id="System.ValueTuple" version="4.4.0" targetFramework="net461" />
   <package id="valueinjecter" version="2.3.3" targetFramework="net451" />
-  <package id="VirtoCommerce.Domain" version="2.25.24" targetFramework="net461" />
+  <package id="VirtoCommerce.Domain" version="2.25.26" targetFramework="net461" />
+  <package id="VirtoCommerce.ExportModule.Core" version="2.1.1" targetFramework="net461" />
+  <package id="VirtoCommerce.ExportModule.Data" version="2.1.1" targetFramework="net461" />
   <package id="VirtoCommerce.Platform.Core" version="2.13.42" targetFramework="net461" />
-  <package id="VirtoCommerce.Platform.Data" version="2.13.37" targetFramework="net461" />
+  <package id="VirtoCommerce.Platform.Data" version="2.13.39" targetFramework="net461" />
```

Comment:
Seems
id="VirtoCommerce.Platform.Core" version="2.13.42"
id="VirtoCommerce.Platform.Data" version="2.13.39"
should be synchronized, probably need to upgrade both 42 (and in project file)